master/internal/db/postgres.go

```diff
@@ -211,7 +211,11 @@ FROM (
                        (SELECT count(*)
                         FROM steps s
                         WHERE s.trial_id = t.id
-                       ) AS num_steps,
+					   ) AS num_steps,
```

Comment:
The github output here suggests that you have but tabs here where there were previously spaces, can you double check that?

Reply:
yep, that was it. oops/thanks.